syntax = "proto2";
package antalk.data;
import "common.proto";
option java_package = "com.antalk.pb";
option cc_generic_services = true;

message GetUserInfoReq {
	optional uint32 saas_id = 1;
	optional string uid = 2;
}

message GetUserInfoRes {
	optional antalk.common.UserInfo user_info = 1;
}

message SetUserStatusReq {
	optional string uid = 1;
	optional int32 dev_type = 2;
	optional string session = 3;
	optional int32 status = 4;
	optional string login_info = 5;
}

message SetUserStatusResp {
	optional antalk.common.ResultType result = 1;
}

service UserStatusService {
      rpc GetUserInfo(GetUserInfoReq) returns (GetUserInfoRes);
      rpc SetUserStatus(SetUserStatusReq) returns (SetUserStatusResp);
};
